### PR TITLE
--standby-for flag for fly machine run|update|clone commands

### PR DESCRIPTION
**Pull request by dangra**

### CODE REVIEW

File: internal/command/machine/clone.go [hunk 1]
```diff
@@ -74,6 +74,10 @@ func newClone() *cobra.Command {
 			Name:        "clear-auto-destroy",
 			Description: "Disable auto destroy setting on new machine",
 		},
+		flag.StringSlice{
+			Name:        "standby-for",
+			Description: "Comma separated list of machine ids to watch for",
```
Comment (kzys) [inline internal/command/machine/clone.go:79]: How about mentioning `source` here?

Comment (dangra) [reply internal/command/machine/clone.go]: very good point, thanks. Added by 747be88e

**Comment by dangra** [comment]
@kzys needs more time or can I get a ✅  to merge and ship asap?

**Review by kzys — APPROVED** [review]
LGTM!

**Pull request merged**